runtime/src/runtime/mod.rs

```diff
@@ -63,12 +65,23 @@ pub trait Runtime: Primitives + Verifier + RuntimePolicy {
     /// The genesis block has epoch zero.
     fn curr_epoch(&self) -> ChainEpoch;
 
+    /// The ID for the EVM-based chain, as defined in https://github.com/ethereum-lists/chains.
+    fn chain_id(&self) -> ChainID;
```

Comment:
I strongly agree with the grumble here, and would like to see this changed. The current flow is that the _client_ (eg. Lotus) knows what EVM chain ID is operational for the blockchain being processed, creates an FVM with this value ("the" chain ID), which then supplies this value through the Runtime to the FEVM's `CHAINID` opcode.

I think it would be in favour of your proposal to add a _Filecoin_ ChainID, and a mapping. The biggest issue is that it requires some "process" work (starting a FIP etc.)

Simpler solutions for now could be:

- Move this value to the Runtime's policy, configure it by actor bundle, and call it `evm_chain_id` (still privileged, but contained to the Runtime without spilling into FVM / clients)
- Keep as is, but rename to `evm_chain_id`

Reply:
This is an Ethereum-specific chain but Ethereum chain IDs have been standardised in [Chain Agnostic](https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md) standards. This should probably read `caip_eip155_chain_id` and it would be less of an offender that way. But the bottom line is that these chain IDs are actually _standardised_ by some body that does have reasonable adoption.

Reply:
Discussed with @Stebalien in standup, he provided some background context. The intention _is_ for these IDs to serve as _Filecoin_ ChainIDs (that are conveniently in the Ethereum namespace). Since we don't have any ChainIDs this works pretty well, and honestly is _probably_ what most other chains do anyway. 

I agree with you that this has not been sufficiently publicized, though -- a footnote in an unrelated FIP is likely not sufficient for this. @jennijuju do you have suggestions here? I don't know if this needs its own FIP, but should be included in a table of networks and chains _somewhere_.

(...we can even eventually include these IDs in messages as replay protection one day...)

Reply:
Is the comment still accurate? Should it be something like "The ID for this chain. Note that Filecoin chain IDs are usually in the Ethereum namespace (i.e. scope "xyz") for [reasons]".

If the comment as it stands _is_ accurate, I still think this should move into some state somewhere (EAM?) rather than be a syscall available to future native actors.